### PR TITLE
Contributing guide docs

### PR DESCRIPTION
**Pull request by mpraglowski**
* How to prepare a new pull request
* Licensing info

### CODE REVIEW

File: railseventstore.org/source/community.html.md [hunk 1]
```diff
@@ -11,3 +11,56 @@ Any kind of contribution is welcomed.
...
+
+* Implement your feature, don't forget about tests & documentation (to see how to work with documentation files check [documentation's readme ](https://github.com/RailsEventStore/rails_event_store/blob/master/railseventstore.org/README.md)
+
+* Make sure your code pass all tests
+
+    ```
+    make test
+    ```
+
+    You could test each project separatelly, just enter the project folder and run tests (`make test` again) there.
```
Comment (joelvh) [inline railseventstore.org/source/community.html.md]: "separately"

File: railseventstore.org/source/community.html.md [hunk 2]
```diff
@@ -11,3 +11,56 @@ Any kind of contribution is welcomed.
...
+
+* Make sure your changes survive mutation testing
+
+    ```
+    make mutate
+    ```
+
+    Will run mutation tests for all projects. The same command executed in specific project's folder will wun mutation tests ony for that project.
```
Comment (joelvh) [inline railseventstore.org/source/community.html.md]: "wun" => "run", "ony" => "only"

File: railseventstore.org/source/community.html.md [hunk 3]
```diff
@@ -11,3 +11,56 @@ Any kind of contribution is welcomed.
...
+    Mutation tests might be time consuming, so you could try to limit the scope of mutations to some specific subjects:
+
+    ```
+    make mutate SUBJECT=code_to_mutate
+    ```
+
+    How to specify `code_to_mutate` is described in [Mutant documentation](https://github.com/mbj/mutant#test-selection).
+
+* Don't forget to [create a Pull Request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/). You could do it even if not eveything is ready. The sooner you will share your changes the quicker feedback you will get.
```
Comment (joelvh) [inline railseventstore.org/source/community.html.md]: "everything"

File: railseventstore.org/source/community.html.md (hunk 1 above)
Comment (paneq) [reply railseventstore.org/source/community.html.md]: Nice findings, isn't it equally fast to pull this branch and fix them?

**Comment by paneq** [comment]
@mpraglowski `[ci skip]` next time you update documentation only :) No need to run hours of tests.

File: railseventstore.org/source/community.html.md (hunk 1 above)
Comment (joelvh) [reply railseventstore.org/source/community.html.md]: @paneq haha - I thought the same, but saw the email fly by and was in the middle of something else. Sorry!! :-)

**Comment by mpraglowski** [comment]
yeah and I've forgot about that again ;) too fast...

**Pull request merged**